utils.pwsh/Invoke-SafeWebRequest.ps1

```diff
@@ -57,7 +57,13 @@ function Invoke-SafeWebRequest {
                 }
             }
 
-            Invoke-External curl --fail --location $(if ( $Env:CI -eq $null ) { '--progress-bar' }) --output $OutFile @HeaderStrings $Uri
+            $ResumeOption = @()
```

Comment:
With now two conditional arguments to `curl`, I wonder if it weren't more readable if we compose one `CurlOptions` object and add all elements at once (including the progress bar and continue option) and then just call `Invoke-External curl @CurlOptions`.

Reply:
That would probably be better. I can do that here or in a separate PR.

Reply:
I'd prefer making the "right" change immediately.

Reply:
I've pushed a change that implements the suggestion in one commit and then adds support for curl resume in another commit.

Reply:
Let me know if you'd like the code adjusted further.